body.tex

```diff
@@ -1,6 +1,136 @@
+
+\subsection{Data visualization, alerting, and analysis}
+
+Enabling users to create their own dashboards and alerts is essential for monitoring and troubleshooting the observatory systems, especially during the current System Integration Testing and Commissioning (SITCOM) phase.
+
+Sasquatch uses Chronograf for data visualization, and Kapacitor for alert management. Dahshboards and alert rules ae defined using InfluxQL, a SQL-like query language with a rich set of functions specific to time series data analysis. Alert notifications are sent to specifc channels on Slack to notify the users about a metric deviation from a predefined threshold or the observatory systems health.
 
-This is the Rubin Observatory overview paper: \cite{2019ApJ...873..111I}.
+Sasquatch is also integrated with the Rubin Science Platform (RSP) \cite{DMTN-082, DMTN-212}. On the RSP at USDF users have access to both real-time and the historical data captured by Sasquatch. This integration enables users to perform data analysis in a JupyterLab environment with Python and leverage tools such as Times Square \cite{SQR-062} for publishing parameterized Jupyter Notebooks.
+
+\section{Sasquatch deployments}
+\label{sec:deploy}
+
+At Rubin Observatory, Sasquatch is deployed to multiple environments, each with different retention periods, storage capacity, and target audience. Sasquatch deployments are managed by Phalanx\footnote{\url{https://phalanx.lsst.io}}, a kubernetes-based platform for deploying and managing application configutation.
+
+Table 1 shows the different environments where we deploy Sasquatch at Rubin Observatory.
+
+\begin{table}[ht]
+    \small
+    \centering
+    \caption{Sasquatch environments at Rubin Observatory}
+    \begin{tabular}{@{}llll@{}}
+        \toprule
+        \textbf{Environment} & \textbf{Retention Period} & \textbf{Current Capacity} & \textbf{Audience} \\
+        \midrule
+        Summit & 30 days & 5 TB & Staff at the Summit \\
+        USDF & Project lifetime & 60 TB & Project at large \\
+        IDF (Google) & 1 year & 30 TB & Project at large (online backup) \\
+        Teststands (Tucson \& Base Facility) & 10 days & 1 TB & Developers \\
+        \bottomrule
+    \end{tabular}
+\end{table}
+
+At the Summit, we store data for 30 days, and the main audience for this deployment is the observatory staff.
+
+At the USDF we have requirements to keep the data for the project lifetime and make it available for the project at large. We also deploy Sasquatch on the Google Cloud Platform where we keep the last year of data as an online backup in case users can't access Sasquatch at the USDF.
+
+Finally, we have independent deployments in Tucson, US and at the Base Facility in La Serena, Chile. At the Teststands we record simulated data, and the main audience for these deployments are our developers.
```

Comment:
It isn't obvious whether the Teststands (should this really be capitalized?) referred to in the 2nd sentence of the paragraph refer to the Tucson and Base Facility deployments, or whether they are introducing yet more deployments. Maybe change the beginning the second sentence from "At the" to "At these",  which makes it explicit that the 2nd sentence is continuing to talk about the deployments mentioned in the first.